SKILL.tsv

```diff
@@ -4586,7 +4586,7 @@ SKILL_20150714_004585	$Increases the time duration of the [Bleeding] status when
 SKILL_20150714_004586	$Bodkin Point: Defense Decrease
 SKILL_20150714_004587	{memo X}[Reduced DEF] of [Bodkin Point] will be reduced by 2% per Attribute level.
 SKILL_20150714_004588	{memo X}The enemies that are shot by [Magic Arrow] will fall into [Silence] with a certain probabillity. The probability will increase by 10% per Attribute level.
-SKILL_20150714_004589	The distance which will maintain [Install Roost] will increase by 20 per Attribute level.
+SKILL_20150714_004589	The range of [Install Roost] will increase by 20 per attribute level.
```

Comment:
Increases the distance that maintains [Install Roost] by 20 per attribute level.

Basically, if you move too far away from it, it will disappear like with anvils.